### PR TITLE
Added CVE-2019-14868

### PR DESCRIPTION
**Pull request by tausif-rh**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14868.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14868",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14868",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14868.json]: Please add reference to:

 https://github.com/att/ast/commit/c7de8b641266bac7c77942239ac659edfee9ecd2

File: 2019/14xxx/CVE-2019-14868.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                {
+                    "vendor_name": "KornShell",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "ksh",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "n/a"
```
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14868.json]: CVE entries must contain version information (8.1.1)

File: 2019/14xxx/CVE-2019-14868.json (hunk 1 above)
Comment (tausif-rh) [reply 2019/14xxx/CVE-2019-14868.json]: Added Reference.

File: 2019/14xxx/CVE-2019-14868.json (hunk 2 above)
Comment (tausif-rh) [reply 2019/14xxx/CVE-2019-14868.json]: Added version information. Thank you

**Comment by cve-team** [comment]
Also be sure to add any version information to the description here as well, thanks!

File: 2019/14xxx/CVE-2019-14868.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14868",
+                "refsource": "CONFIRM"
+            }
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A flaw was found in the way ksh evaluates certain environment variables. An attacker could use this flaw to override or bypass environment restrictions to execute shell commands. Services and applications that allow remote unauthenticated attackers to provide one of those environment variables could allow them to exploit this issue remotely."
```
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14868.json]: Please include the affected version of ksh in the description.

**Pull request merged**